### DISCUSSION TITLE
Cobia

### DISCUSSION THREAD

**Discussion by muddro1** [Jip-Hop/jailmaker#34] [General]
Has this been tried on cobia?

**Comment by Jip-Hop** [comment]
Cobia hasn't been released but I have tried this method on one of the nightly builds not too long ago without any problems. I recommend you to try it yourself inside a VM.

**Comment by spusuf** [comment]
Yes it works on both BETA1 and RC1 (release candidate) so I doubt they'd change anything significantly at this point for full release.

**Comment by PackElend** [comment]
so we can be optimistic that it will gonna make it into the official release and service at least its successor arrives 🤞

**Reply by spusuf** [reply]
If by "official release" you mean integration into the TrueNAS GUI then it's highly unlikely at this point, the TrueNAS team has bigger things to work on and the feature request did not get a lot of upvotes.

**Reply by PackElend** [reply]
that would be great but I only meant that is still fully functional when Copia is released.

by the way, whichtrequest do you mean; may I missed that

**Reply by spusuf** [reply]
There was a feature request on the TrueNAS forums/discussion site. But there will be no "official" cobia release because everything is already compatible.

**Reply by ericderace** [reply]
I just upgraded from bluefin to Cobia, no issues for me. Thanks for the answers

**Comment by ericderace** [comment]
@spusuf did you upgrade from bluefin to cobia, and no issues whatsoever with jlmkr?

**Reply by muddro1** [reply]
Had no issues here

**Reply by spusuf** [reply]
Can confirm, no issues. running 20 jails without issue. Some Debian 11, some Debian 12.
